lib/matplotlib/tests/test_axes.py

```diff
@@ -4946,6 +4946,11 @@ def test_ls_ds_conflict():
         plt.plot(range(32), linestyle='steps-pre:', drawstyle='steps-post')
 
 
+@image_comparison(baseline_images=['bar_uint8'], extensions=['png'])
```

Comment:
Any chance this and the test below could be written in a way that allows reusing other test images in order to avoid further bloating the repo?